index.html

```diff
@@ -5,14 +5,16 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Hekto</title>
+	<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100,200,300,regular,500,600,700,100italic,200italic,300italic,italic,500italic,600italic,700italic" rel="stylesheet" />
+	<link href="https://fonts.googleapis.com/css?family=Lato:100,100italic,300,300italic,regular,italic,700,700italic,900,900italic" rel="stylesheet" />
 	<link rel="stylesheet" href="./styles/style.css">
 </head>
 
 <body>
 	<header class="site-header">
 
 		<div class="header-top">
-			<div class="container">
```

Comment:
Why did you do it? We have a class container in our CSS, which we're using across all sections. 

Bring back container class